README.md

```diff
@@ -15,27 +16,27 @@ A user-friendly GUI that generates HTML/PHP pages with an identical style, but t
 - [X] Submit Formation of Project Teams
 - [X] Submit Github Repository
 - [X] Read [Github README Formatting](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) should you want to work on the README, makes it easier to understand what does what.
-- [ ] Submit Project Vision for the Course Project
+- [X] Submit Project Vision for the Course Project
 - [ ] Submit Project Team Roles
 - [ ] Submit Product Backlog
 
 </details>
 
  
-## Product Vision
+## Current Product Vision
```

Comment:
I won't keep a log of all Product Visions on the readme, so "current" is probably not the best change.